.github/workflows/build.yml

```diff
@@ -33,3 +33,56 @@ jobs:
               uses: matlab-actions/run-tests@v1
               with:
                  select-by-folder: opentelemetry-matlab/test
+    build-and-run-tests-windows:
+        runs-on: windows-latest
+        env:
+            OPENTELEMETRY_MATLAB_INSTALL: "${{ github.workspace }}/otel_matlab_install"
+        steps:
+            - name: Download OpenTelemetry-Matlab source
+              uses: actions/checkout@v3
+              with: 
+                path: opentelemetry-matlab
+            - name: Install MATLAB
+              uses: matlab-actions/setup-matlab@v1
+              with: 
+                release: R2023a
```

Comment:
Is it necessary to pin MATLAB to a specific release? Or can we just let it use the latest?